### DISCUSSION TITLE
Proxy Host Categories!

### DISCUSSION THREAD

**Discussion by bendini20** [NginxProxyManager/nginx-proxy-manager#1314] [Ideas]
It would be very nice to be able to categorize hosts. Whether that be with a tag or with FOLDERS. For example, I would like to have 1 category for html code-hosted sites and one for a certain docker. It would be much easier to navigate to where to you want to go with folders.

You could add an item to a category or folder via the same settings were you add the domain and host, or in the handburger menu where you can also edit, disable, or delete.

**Comment by jc21** [comment]
I like this idea :)

**Reply by bendini20** [reply]
Thanks. Would be very helpful since I am already up to 11 hosts and 2 separate domains. Would like to categorize my domains!

**Reply by jc21** [reply]
Those are rookie numbers. You gotta bump those numbers up!

**Reply by bendini20** [reply]
Ha! Maybe if I get gigabit fiber and have the confidence to serve many domains..

**Comment by chaptergy** [comment]
There is an open issue suggesting this, but it also includes a search bar as an alternative. I've edited the issue to include it more explicitly in the title: https://github.com/jc21/nginx-proxy-manager/issues/409.

**Comment by gregfr** [comment]
Greetings!
Any news on this? I'm using NPM to manager more than a hundred sites on several installation, and it would be a great help to be able to tag/categorize, to know which sites is served through which proxy...
Regards
